### DISCUSSION TITLE
Best way to add a feed from Reddit?

### DISCUSSION THREAD

**Discussion by tomhusband** [FreshRSS/FreshRSS#6558] [Q&A] [answered]
I'm moving away from Inoreader to FreshRSS and have already imported all my feeds. I'm now adding a few more. In Inoreader I could just paste the link, like https://new.reddit.com/r/selfhosted/, and Inoreader would get the feed. In FreshRSS using the default I get an error and it can't be found. There are several pull down options in FreshRSS so it's probably one of those. Do I need to change the link address?

In trying various links the log shows errors like:
HTML+XPath Web scraping failed for [https://new.reddit.com/r/selfhosted/]
cURL error 22: The requested URL returned error: 404 [https://new.reddit.com/r/selfhosted/feed/]
cURL error 22: The requested URL returned error: 404 [https://new.reddit.com/r/selfhosted/force_feed]

**Comment by Alkarex** [comment, marked as answer]
Try https://www.reddit.com/r/selfhosted/.rss
See https://www.reddit.com/wiki/rss/
